modules/integration-tests/src/main/java/org/apache/fluo/integration/impl/StochasticBankIT.java

```diff
@@ -62,8 +60,8 @@
   @Test
   public void testConcurrency() throws Exception {
 
-    conn.tableOperations().setProperty(table, Property.TABLE_MAJC_RATIO.getKey(), "1");
-    conn.tableOperations().setProperty(table, Property.TABLE_BLOCKCACHE_ENABLED.getKey(), "true");
+    conn.tableOperations().setProperty(table, "table.compaction.major.ratio", "1");
```

Comment:
The following file has constants for Accumulo props that Fluo uses.

./modules/accumulo/src/main/java/org/apache/fluo/accumulo/util/AccumuloProps.java

Reply:
Oh, cool. I didn't know about that. I can change to use that, since it can help avoid bugs due to typos.